theme/static/style/style.css

```diff
@@ -446,7 +567,34 @@ pre code {
     /*
         For mobile phones - only override styles which
         are different than the tablet section! No duplication!
-    */
+    */  
+    
+    /*navigation section*/
+    .topnav .col-4 img{
+        margin-top: 12.5px;
+        max-width: 200px;
+    }
+    .topnav .col-8 li{
+        display: none;
+    }
+    .topnav .col-8 li:last-child{
+        display: block;
+    }
+    .topnav .open-menu{
+        display: inline-block;
+    }
+
+    /*header section*/
+    .header .col-7{
+        display: none;
+    }
+    .header .col-5{
+        width: 100%;
+    }
+    .header .header-btns{
+        margin-top: 50px;
```

Comment:
An equivalent to this property exists on line ``248``, so there is no use for this one.